app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,10 @@
+class RestaurantsController < ApplicationController
+
+	def index
+		@restaurants = Restaurant.all
+	end
+
+	def show 
+		@restaurant = Restaurant.find(params[:id])
+	end
+end
```

Comment:
This may just be github, but the indentation is a littly wonky here

Reply:
Not seeing any way for us to fix this, looks the same on both of our editors